File: solutions/3373-maximum-prime-difference/solution.py
class Solution(object):
    def maximumPrimeDifference(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        def isprime(n):
            if n<2:
                return False
            for i in range(2, int(n ** 0.5) + 1):
                if n%i==0:
                    return False
            return True

        ls=[]
        s=len(nums)
        for i in range(0,s):
            if isprime(nums[i]):
                ls.append(i)
        return max(ls)-min(ls)

File: solutions/3373-maximum-prime-difference/solution.java
class Solution {
    public int maximumPrimeDifference(int[] nums) {
        int b=0,c=0;
        for(int i=0;i<nums.length;i++)
        {
            if(ss(nums[i]))
            {
                b=i;
                break;
            }
        }
        for(int i=nums.length-1;i>=0;i--)
        {
            if(ss(nums[i]))
            {
                c=i;
                break;
            }
        }
        return Math.abs(c-b);
    }
    public boolean ss(int n)
    {
        if(n<2)
           return false;
        for(int i=2;i<(int)(Math.sqrt(n)+1);i++)
            if(n%i==0)
                return false;
       return true;
    }
}
